contracts/cw3-fixed-multisig/src/state.rs

```diff
@@ -176,16 +184,21 @@ pub struct Ballot {
     pub vote: Vote,
 }
 
+type ProposalID = u64;
+type VoterWeight = u64;
+
 // unique items
-pub const CONFIG: Item<Config> = Item::new("config");
-pub const PROPOSAL_COUNT: Item<u64> = Item::new("proposal_count");
+pub static CONFIG: Item<Config, Json> = Item::new(b"config");
+pub static PROPOSAL_COUNT: Item<u64, Json> = Item::new(b"proposal_count");
 
-// multiple-item map
-pub const BALLOTS: Map<(u64, &Addr), Ballot> = Map::new("votes");
-pub const PROPOSALS: Map<u64, Proposal> = Map::new("proposals");
+// binary search tree (heap)
```

Comment:
need a more clear comment, maybe
```suggestion
// keys for `VOTERS` map
```

Reply:
Done.